service_extensions/callouts/add_header/requirements.txt

```diff
@@ -0,0 +1,2 @@
+grpcio==1.59.0
+grpcio-tools==1.59.0
```

Comment:
do we hvae to pin this to these versions? Given that the files in Pr are generated using debian tooling, I am not sure if the python version is same as debian

Reply:
is this same as debian bullseye grpc tools?

Reply:
There are no dedicated OS specific src/codes as per https://github.com/grpc/grpc. Its all common source code. So, yes.